apps/web/pages/settings/platform/oauth-clients/[clientId]/edit/webhooks.tsx

```diff
@@ -0,0 +1,107 @@
+import { useParams } from "next/navigation";
+import { useRouter } from "next/navigation";
+
+import Shell from "@calcom/features/shell/Shell";
+import { WebhookForm } from "@calcom/features/webhooks/components";
+import { showToast } from "@calcom/ui";
+
+import {
+  useOAuthClientWebhooks,
+  useCreateOAuthClientWebhook,
+  useUpdateOAuthClientWebhook,
+} from "@lib/hooks/settings/platform/oauth-clients/useOAuthClientWebhooks";
+
+import PageWrapper from "@components/PageWrapper";
+import NoPlatformPlan from "@components/settings/platform/dashboard/NoPlatformPlan";
+import { useGetUserAttributes } from "@components/settings/platform/hooks/useGetUserAttributes";
+
+export default function EditOAuthClientWebhooks() {
+  const router = useRouter();
+  const params = useParams<{ clientId: string }>();
+  const clientId = params?.clientId || "";
+
+  const { isUserLoading, isPlatformUser, isPaidUser } = useGetUserAttributes();
+
+  const {
+    data: webhooks,
+    status: webhooksStatus,
+    isFetched: isWebhooksFetched,
+    refetch: refetchWebhooks,
+  } = useOAuthClientWebhooks(clientId);
+  const webhookId = webhooks?.[0]?.id ?? "";
+  const webhook = webhooks?.[0];
+  const { mutateAsync: createWebhook } = useCreateOAuthClientWebhook(clientId);
+  const { mutateAsync: updateWebhook } = useUpdateOAuthClientWebhook(clientId);
+
+  if (isUserLoading) return <div className="m-5">Loading...</div>;
+
+  if (isPlatformUser && isPaidUser) {
```

Comment:
If `isPlatformUser` but `!isPaidUser` we could describe a banner - "Upgrade to ESSENTIALS plan to access webhooks"

Reply:
IsPaidUser just checks if the user has billing, but yeah I agree if we start gating features we should